src/ObjectProxy.php

```diff
@@ -7,6 +7,8 @@
 use Throwable;
 
 /**
+ * @internal
```

Comment:
[Here](https://github.com/yiisoft/proxy#custom-base-proxy-class) used `ObjectProxy`. Are you sure that is internal class?

Reply:
It's not. Fixed, thanks.